rpm/xcache.spec

```
@@ -1,6 +1,6 @@
 Name:      xcache
 Summary:   XCache scripts and configurations
-Version:   1.1.0
+Version:   1.1.1
 Release:   0.1%{?dist}
```

It looks like I forgot to update the spec file for the second RC (https://github.com/opensciencegrid/xcache/releases/tag/v1.1.0-0.2)

```suggestion
Release:   0.3%{?dist}
```